### ISSUE TITLE
[bug]: cli defaulting to TypeScript but then defaults to tailwind.config.js

### ISSUE THREAD

**Issue by greghesp** [shadcn-ui/ui#4467]
### Describe the bug

_I'm using Remix in this example, but I suspect this causes issues with other frameworks too_

When running `npx shadcn-ui@latest init`, it takes you through the CLI setup process, defaults to using TypeScript, but then for tailwind, it defaults to tailwind.config.js rather than a .ts file.  

I suspect this is the cause for issue [2704](https://github.com/shadcn-ui/ui/issues/2704)

![image](https://github.com/user-attachments/assets/35ca1432-eb4a-4159-8c81-ebd43b7ddb36)

### Affected component/components

All

### How to reproduce

Follow the docs here: https://ui.shadcn.com/docs/installation/remix

After step 3, you will have a `tailwind.config.js` and a `tailwind.config.ts` file which conflict

### System Info

```bash
n/A
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by shadcn** [comment]
This issue has been automatically marked as stale due to one year of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you. (This is an automated message)

**Comment by shadcn** [comment]
This issue has been automatically closed due to one year of inactivity. If you’re still experiencing a similar problem or have additional details to share, please open a new issue following our current issue template. Your updated report helps us investigate and address concerns more efficiently. Thank you for your understanding! (This is an automated message)
